Project.toml

```
@@ -1,17 +1,16 @@
 name = "Setfield"
 uuid = "efcf1570-3423-57d1-acb7-fd33fddbac46"
-version = "1.0.0"
+version = "1.0.1"
```

I am no semver lawyer, but I think this is more than a patch release, if we bump the julia version, see below.
```suggestion
version = "1.1.0"
```